management-api-guide.md

```diff
@@ -289,9 +291,9 @@ Here is a sample request to
   },
   "salesUnitName": "ACME Fantastic Fitness",
   "salesUnitLogo": "VGhlIGltYWdlIGdvZXMgaGVyZQ==",
-    "settlementAccountNumber": {
-    "scheme": "BBAN:NO",
-    "id": "86011117947"
+    "bankAccount": {
```

Comment:
The Report API uses `payoutBankAccount`, so I think we should prefix the account here too, and keep `settlementAccount`.

Report API: https://developer.vippsmobilepay.com/api/report/#/paths/~1settlement~1v1~1ledgers/get

Reply:
Like this? [91a1481](https://github.com/vippsas/management-api/pull/45/commits/91a1481cd8fd45a463d92d974450d4c1a15150a4)

Reply:
So close!

Reply:
I see your edit in the first post there, no sneaking that past me, your fault! :P